apollo-router-core/src/json_ext.rs

```diff
@@ -35,94 +25,13 @@ pub trait ValueExt {
     /// values in `self`.
     #[track_caller]
     fn is_subset(&self, superset: &Value) -> bool;
+
+    /// create a Value by inserting a value at a subpath
```

Comment:
:screwdriver: please spell things properly in doc.

```suggestion
    /// Create a `Value` by inserting a value at a subpath.
```

Reply:
right, thanks